modelkit/core/models/tensorflow_model.py

```diff
@@ -34,11 +34,45 @@
     from tensorflow_serving.apis.get_model_metadata_pb2 import GetModelMetadataRequest
     from tensorflow_serving.apis.predict_pb2 import PredictRequest
 
-except ModuleNotFoundError:
+except ModuleNotFoundError:  # pragma: no cover
     logger.info("Tensorflow serving is not installed")
 
 
-class TensorflowModel(Model[ItemType, ReturnType]):
+class AbstractTensorflowModel:
+    output_shapes: Dict[str, Tuple]
+    output_dtypes: Dict[str, Type]
+    output_tensor_mapping: Dict[str, str]
+
+    def _is_empty(self, item) -> bool:
```

Comment:
```suggestion
    def _is_empty(self, item: Any) -> bool:
```

Reply:
This would cause an error, because we pass `item`s that have a correctly defined type in actual `TensorFlowModel`s